x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/factories/utils/ecs_types_validators/is_valid_keyword.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { SearchTypes } from '../../../../../../../common/detection_engine/types';
```

Comment:
@e40pud , with the latest changes in mappings, I don't think we need this check anynore